.github/workflows/docs-pages.yaml

```diff
@@ -0,0 +1,41 @@
+name: "Docs / Publish"
```

Comment:
Why the naming change from `docs-pages@v2` to `docs-pages`?

Reply:
We received feedback from other projects they prefer the workflows files not to be versioned. Therefore, we applied this recommendation in the main repo https://github.com/scylladb/sphinx-scylladb-theme

https://github.com/scylladb/sphinx-scylladb-theme/pull/348